lib/ws_servers/api/handlers/on_algo_order_submit.js

```diff
@@ -21,9 +21,12 @@ module.exports = async (server, ws, msg) => {
     return sendError(ws, 'Unauthorized')
   } else if (exID !== 'bitfinex') {
     return sendError(ws, 'Algo orders currently only enabled for Bitfinex')
-  } else if (!ws.aoc) {
-    return sendError(ws, 'Unauthorized')
   }
 
-  ws.aoc.startAO(exID, aoID, packet)
+  try {
+    await ws.algoWorker.submitOrder(aoID, packet)
+    this.send(['submit', this.userID, exID, aoID, packet])
```

Comment:
newbie question: I don't see the definition for `this.send` in this context and, with the lack of tests we can not assure that this will break in runtime. Is this implicitly inherited from somewhere else?

Reply:
yeah this is the old code that was used to call algo server which we dont need anymore ..

Reply:
whoops! :)

Reply:
fixed!